packages/pgadmin4-desktop-deb/pgadmin4-desktop-deb.pacscript

```diff
@@ -0,0 +1,14 @@
+pkgname="pgadmin4-desktop-deb"
+repology=("project: pgadmin4-desktop")
+arch=('amd64')
+gives="pgadmin4-desktop"
+pkgver="9.0"
+source=("https://ftp.postgresql.org/pub/pgadmin/pgadmin4/apt/oracular/dists/pgadmin4/main/binary-amd64/pgadmin4-desktop_${pkgver}_amd64.deb")
+source_noble=("https://ftp.postgresql.org/pub/pgadmin/pgadmin4/apt/noble/dists/pgadmin4/main/binary-amd64/pgadmin4-desktop_${pkgver}_amd64.deb")
+makedepends=("apparmor-profiles")
+pacdeps=("pgadmin4-server-deb")
+pkgdesc="The desktop user interface for pgAdmin"
```

Comment:
https://github.com/pacstall/pacstall/wiki/101.1-Variables#pkgdesc

Look at the rules.